lib/server/db/seeds/connection.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Utility for adding configurable Connections seeds to the
+ * database.
+ *
+ * @module ConnectionSeed
+ */
+var User = require("../model/User");
+var Connection = require("../model/Connection");
+
+/**
+ * @typedef {Object} ConnectionSeedConfig
+ * @property {String} ConnectionSeedConfig.user - The from-User
+ * @property {String[]} ConnectionSeedConfig.friends - The to-Users
+ */
+
+/**
+ * Create connections between a user and a variable number of friends and
+ * returns a then-able Promise, passing the connections array to the next "then".
+ *
+ * @private
+ * @function createConnections
+ * @param {ConnectionSeedConfig} config - The name of the from-User
+ * @returns {Promise}
+ */
+function createConnections(config) {
+    return User
+        .findOne({
+            username: config.user
+        })
+        .then(function (user) {
+            return User
+            // find all the input users
+                .find({
+                    $or: config.friends.map(function (friend) {
+                        return {
+                            username: friend
+                        };
+                    })
+                })
+                // bundle up the user and friends to pass as data
+                .then(function (friends) {
+                    return {
+                        user: user,
+                        friends: friends
+                    };
+                });
+        })
+        .then(function (data) {
+            return data.friends.map(function (friend) {
+                return new Connection({
+                    user: data.user._id,
+                    friend: friend._id
+                });
+            });
+        })
+        .catch(console.error);
```

Comment:
Catches happen when an error is thrown inside the `Promise`. The input function to the `catch` method expects a single parameter, which is an `Error` object. Since `console.error` takes an `Error` as a parameter, we can concisely just pass that function to `catch`. 

In the app itself, we want to handle the error and degrade gracefully. But in the seeds file, we just want to *know* what the error is so logging it out should suffice.